Dockerfile

```diff
@@ -18,6 +18,7 @@ ENV JAVA_HOME=/usr/lib/jvm/java-17-openjdk \
 WORKDIR /opt
 
 RUN set -eux; \
+    chmod -R 666 /etc/ssl/certs/java/cacerts; \
```

Comment:
* I think you should do the chmod after installing the `openjdk17-jre`.
* `entrypoint.sh` is trying to import the certificate in `/usr/lib/jvm/default-jvm/jre/lib/security/cacerts` which is a symlink to `/etc/ssl/certs/java/cacerts`. For clarity, I would change entrypoint.sh to use the same path.
* I wonder if we are not defeating the purpose of using a non-root account by letting everyone modify this file. But I don't have a better idea at the moment.

Reply:
Right. I moved the chmod statement to after the installation of openjdk17-jre and updated the `entrypoint.sh` to align.

For the non-root account; yes. I'm not exactly sure what is gained by dropping to a non-root account in the docker image, so I can't argue if it's something that should be kept or removed.